atlite/datasets/era5.py

```diff
@@ -338,36 +340,76 @@ def retrieve_data(product, chunks=None, tmpdir=None, lock=None, **updates):
     If you want to track the state of your request go to
     https://cds-beta.climate.copernicus.eu/requests?tab=all
     """
-    request = {"product_type": "reanalysis", "format": "netcdf"}
+
+    # Set url for data download, this allows to switch to different data
+    # sources more easily.
+    url = "https://cds.climate.copernicus.eu/api"
+
+    request = {
+        "product_type": ["reanalysis"],
+        "data_format": "netcdf",
+        "download_format": "zip",
+    }
+
     request.update(updates)
 
     assert {"year", "month", "variable"}.issubset(
         request
     ), "Need to specify at least 'variable', 'year' and 'month'"
 
     client = cdsapi.Client(
-        info_callback=logger.debug, debug=logging.DEBUG >= logging.root.level
+        url=url, info_callback=logger.debug, debug=logging.DEBUG >= logging.root.level
     )
     result = client.retrieve(product, request)
 
     if lock is None:
         lock = nullcontext()
 
     with lock:
-        fd, target = mkstemp(suffix=".nc", dir=tmpdir)
+        fd, target_zip = mkstemp(suffix=".zip", dir=tmpdir)
```

Comment:
Do you have any experience on whether zipping always, even if `.nc' files could be downloaded, has any performance drawbacks?
I could imagine that it might even improve it, since the bottleneck is most likely download bandwidth, but I do not know.

Reply:
My experience was that downloading .zip files directly, improves performance. Nevertheless, I did not test it.

Reply:
Which makes sense. That's good enough